sql/bgw_scheduler.sql

```diff
@@ -21,13 +21,21 @@ INSERT INTO _timescaledb_config.bgw_job (id, application_name, job_type, schedul
 (1, 'Telemetry Reporter', 'telemetry_and_version_check_if_enabled', INTERVAL '24h', INTERVAL '100s', -1, INTERVAL '1h')
 ON CONFLICT (id) DO NOTHING;
 
-CREATE OR REPLACE FUNCTION add_drop_chunks_policy(
+-- Add a retention policy to a hypertable or continuous aggregate.
+-- The retention_window (typically an INTERVAL) determines the 
+-- window beyond which data is dropped at the time
+-- of execution of the policy (e.g., '1 week'). Note that the retention 
+-- window will always align with chunk boundaries, thus the window 
+-- might be larger than the given one, but never smaller. In other
+-- words, some data beyond the retention window 
+-- might be kept, but data within the window will never be deleted.
```

Comment:
Nit (on my own suggestion):

```suggestion
-- Add a retention policy to a hypertable or continuous aggregate.
-- The retention_window (typically an INTERVAL) determines the 
-- window (e.g., '1 week') beyond which data is dropped at the time
-- of execution of the policy . Note that the retention 
-- window will always align with chunk boundaries, thus the window 
-- might be larger than the given one, but never smaller. In other
-- words, some data beyond the retention window 
-- might be kept, but data within the window will never be deleted.
```